changelog/unreleased/kong/bump-atc-router.yml

```diff
@@ -0,0 +1,3 @@
+message: Bumped atc-router from 1.2.0 to 1.4.0
```

Comment:
```suggestion
message: Bumped atc-router from 1.3.1 to 1.4.0
```

Reply:
1.3.1 was unreleased, we should use the last released version here which is `1.2.0`:

https://github.com/Kong/kong/blob/3.5.0/.requirements